### DISCUSSION TITLE
Use Key Store within inventory

### DISCUSSION THREAD

**Discussion by carokiel** [semaphoreui/semaphore#2100] [Q&A]
Hi,
we need to use the microsoft.ad.ldap.yml inventory file to dynamically create our inventory.
However, this needs a username and password inside. I can use environments variables, but that means; i store the password in clear text.
Is there a way (most probably is and I just don't know) to use the Key Store within those variables?

Really appreciate any help, here.
All the best,
Caroline

**Comment by fiftin** [comment]
Hi @carokiel 

Currently it is not possible because I have no idea how to pass secret securely to `ansible-playbook`.  @tboerger may be you have idea? It is third time when asking this feature :)
